src/PostgREST/QueryBuilder.hs

```diff
@@ -429,9 +429,9 @@ pgFmtFilter table (Filter fld (OpExpr hasNot oper)) = notOp <> " " <> case oper
 
    Fts mode lang val ->
      pgFmtFieldOp "fts" <> " " <> case mode of
-       Normal -> "to_tsquery("
-       Plain  -> "plainto_tsquery("
-       Phrase -> "phraseto_tsquery("
+       Just Plain  -> "plainto_tsquery("
+       Just Phrase -> "phraseto_tsquery("
+       Nothing     -> "to_tsquery("
```

Comment:
It seems counter-intuitive to model this as `Maybe FtsMode` rather than simply `FtsMode` since even if the user omits an explicit mode we're still choosing some kind of a mode.